.github/COMMITTING.md

```diff
@@ -0,0 +1,39 @@
+# Committing to Nemo
+
+## Becoming a Committer
+
+To get started contributing to Nemo, refer to the [contributor's guide](CONTRIBUTING.md).
+
+Based on the contributions of the contributors for Nemo, PMC regularly adds new committers from the active contributors.
+The qualifications for new committers include
+
+1. Sustained contributions to Nemo
+2. Quality of contributions
+3. Community involvement
+
+The type and level of contributions considered may vary by project area - code itself, documentations, platform support for specific OSes, storage systems, etc.
+
+## How to Merge a Pull Request
```

Comment:
Add `Update branch` case ?

Reply:
You mean with the master right? I could add that up in a sec.

Reply:
Done